docs/user/autoyast/reference.md

```diff
@@ -0,0 +1,402 @@
+| `wireless_default_key`         | no        |                                      |                                                  |
+| `wireless_eap_auth`            | no        |                                      |                                                  |
+| `wireless_eap_mode`            | no        |                                      |                                                  |
+| `wireless_essid`               | yes       | `ssid`                               |                                                  |
+| `wireless_frequency`           | no        |                                      |                                                  |
+| `wireless_key`                 | no        |                                      |                                                  |
+| `wireless_key_0`               | no        |                                      |                                                  |
+| `wireless_key_1`               | no        |                                      |                                                  |
+| `wireless_key_2`               | no        |                                      |                                                  |
+| `wireless_key_3`               | no        |                                      |                                                  |
+| `wireless_key_length`          | no        |                                      |                                                  |
+| `wireless_mode`                | yes       | `mode`                               |                                                  |
+| `wireless_nick`                | no        |                                      |                                                  |
+| `wireless_nwid`                | no        |                                      |                                                  |
+| `wireless_peap_version`        | no        |                                      |                                                  |
+| `wireless_power`               | no        |                                      |                                                  |
+| `wireless_wpa_anonid`          | no        |                                      |                                                  |
+| `wireless_wpa_identity`        | no        |                                      |                                                  |
+| `wireless_wpa_password`        | yes       | `password`                           |                                                  |
+| `wireless_wpa_psk`             | yes       | `password`                           |                                                  |
+
+## services-manager
+
+There are plans to support this section in the future. You can use a post-installation script to handle these cases.
```

Comment:
From these sentences it may not be clear whether post-installation is a final solution. I would add "meanwhile" somewhere to make it clear.